soco/core.py

```diff
@@ -1388,31 +1388,28 @@ def add_multiple_to_queue(self, items, container=None):
             items (list): A sequence of items to the be added to the queue
             container (DidlObject, optional): A container object which
                 includes the items.
-
-        Returns:
-            int: The index at which the items were enqueued.
         """
-        uris = ' '.join([item.resources[0].uri for item in items])
-        uri_metadata = ' '.join([to_didl_string(item) for item in items])
         if container is not None:
             container_uri = container.resources[0].uri
             container_metadata = to_didl_string(container)
         else:
             container_uri = ''
             container_metadata = ''  # pylint: disable=redefined-variable-type
-        response = self.avTransport.AddMultipleURIsToQueue([
-            ('InstanceID', 0),
-            ('UpdateID', 0),
-            ('NumberOfURIs', len(items)),
-            ('EnqueuedURIs', uris),
-            ('EnqueuedURIsMetaData', uri_metadata),
-            ('ContainerURI', container_uri),
-            ('ContainerMetaData', container_metadata),
-            ('DesiredFirstTrackNumberEnqueued', 0),
-            ('EnqueueAsNext', 0)
-        ])
-        qnumber = response['FirstTrackNumberEnqueued']
-        return int(qnumber)
+        csize = 16
```

Comment:
So in the nitpicking category.

We usually prefer to write variables out, unless they are only used on the same line. The loop expression maybe also got a bit complex. How about:
```python
chunk_size = 16
for index in range(0, len(items), chunk_size):
    chunk = items[index, index + chunk_size]